lib/cdo/mailjet.rb

```diff
@@ -0,0 +1,110 @@
+require 'mailjet'
+require 'honeybadger/ruby'
+require_relative './shared_constants/mailjet_constants'
+
+module MailJet
+  include MailJetConstants
+
+  API_KEY = CDO.try(:mailjet_api_key).freeze
+  SECRET_KEY = CDO.try(:mailjet_secret_key).freeze
+
+  def self.enabled?
+    return false unless DCDO.get('use_mailjet', false)
```

Comment:
I always have a hard time reading 'unless', but I'm surprised ust_mailjet would be false here

Reply:
This feels super unintuitive and requires a few mental jumps. Why can't we just use the much simpler form?

```suggestion
    return DCDO.get('use_mailjet', false)
```

Reply:
Combined this line with the one below

Reply:
And used `present?` over negating `nil?`